lib/chip/service_exception.rb

```diff
@@ -8,7 +8,7 @@ class ServiceException < Common::Exceptions::BackendServiceException
 
     UNMAPPED_KEY = 'unmapped_service_exception'
 
-    def initialize(key: nil, response_values: {}, original_status: nil, original_body: nil)
+    def initialize(key, response_values = {}, original_status = nil, original_body = nil)
```

Comment:
Why move away from kwargs here?

Reply:
because the base class is calling the initialize on module's `ServiceException` without the named args: https://github.com/department-of-veterans-affairs/vets-api/blob/9986d24dd0bf02de47239891e828045292d5d397/lib/common/client/base.rb#L110

I guess that'll need to be changed for the sub-classes to have kwargs in their constructors.